qa/tasks/salt_manager.py

```diff
@@ -207,6 +208,32 @@ def master_rpm_q(self, pkg_name):
             installed = False
         return installed
 
+    def __get_rpm_pkg_version(self, pkg):
+        """Gather RPM package version"""
+        version = None
+        try:
+            r = self.master_remote.run(args=[
```

Comment:
this line should run on the `remote` object passed into the function, not on `self.master_remote`